src/OSPSuite.FuncParser/ParsedFunction.cs

```diff
@@ -8,11 +8,10 @@ namespace OSPSuite.FuncParser
 {
    internal static class FuncParserImportDefinitions
    {
-
-#if _WINDOWS
-      public const String NATIVE_DLL = "OSPSuite.FuncParserNative.dll";
+#if NETSTANDARD2_0
+      public const String NATIVE_DLL = "OSPSuite.FuncParserNative";
 #else
-      public const String NATIVE_DLL = "libOSPSuite.FuncParserNative.so";
+      public const String NATIVE_DLL = "OSPSuite.FuncParserNative.dll";
```

Comment:
Under .NET Framework the user of this string `DllImport` only searches the exact file name specified, and that name has to be a compile time constant.

After that, a convention was introduced to search by the conventional names for shared libraries on each platform. So you only have to use OSPSuite.FuncParserNative and on Windows the library `OSPSuite.FuncParserNative.dll` will be loaded and on Linux `libOSPSuite.FuncParserNative.so`.

Since we cannot use the convention in net472, we still need separate .NET assemblies, but this way nuget can manage which version is resolved based on the top-line project.

For PK-Sim, targeting .net472 gives us line 14, for OSPSuite-R under r-Sharp, we get line 12.